presidio-analyzer/presidio_analyzer/protobuf_models/analyze_pb2_grpc.py

```diff
@@ -1,7 +1,7 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 import grpc
 
-import analyze_pb2 as analyze__pb2
+from . import analyze_pb2 as analyze__pb2
```

Comment:
was this regenerated?

Reply:
just saw you used sed to alter these files

Reply:
why is this a must. I'd prefer to have as little manual intervention as possible

Reply:
No, I used sed command that's in the comment